docs-src/python/how-to-set-a-dynamic-activity-in-python.md

```diff
@@ -0,0 +1,49 @@
+---
+id: how-to-set-a-dynamic-activity-in-python
+title: How to set a Dynamic Activity
+sidebar_label: Set a Dynamic Activity
+description: Use `dynamic=True` on the `@activity.defn` decorator to make an Activity dynamic.
+tags:
+- dynamic activity
+- python sdk
+- code sample
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-python/blob/dynamic-ent/dynamic_entities/your_dynamic_activity_dacx.py. -->
+
+A Dynamic Activity in Temporal is an Activity that is invoked dynamically at runtime if no other Activity with the same name is registered.
+An Activity can be made dynamic by adding `dynamic=True` to the `@activity.defn` decorator.
+You must register the Activity with the [Worker](https://python.temporal.io/temporalio.worker.html) before it can be invoked.
+
+The Activity Definition must then accept a single argument of type `Sequence[temporalio.common.RawValue]`.
+The [payload_converter()](https://python.temporal.io/temporalio.activity.html#payload_converter) function is used to convert a `RawValue` object to the desired type.
+
+<a class="dacx-source-link" href="https://github.com/temporalio/documentation-samples-python/blob/dynamic-ent/dynamic_entities/your_dynamic_activity_dacx.py">View source code</a>
+
+```python
+# ...
+@activity.defn(dynamic=True)
+async def dynamic_greeting(args: Sequence[RawValue]) -> str:
+    arg1 = activity.payload_converter().from_payload(args[0].payload, YourDataClass)
+    return (
+        f"{arg1.greeting}, {arg1.name}!\nActivity Type: {activity.info().activity_type}"
+    )
+# ...
+async def main():
+    client = await Client.connect("localhost:7233")
+
+    async with Worker(
+        client,
+        task_queue="dynamic-activity-task-queue",
+        workflows=[GreetingWorkflow],
+        activities=[dynamic_greeting],
+    ):
+        result = await client.execute_workflow(
```

Comment:
I know we do in some samples, but arguably we don't want to combine client use and worker use in the same snippet (can probably get rid of client use altogether and just show worker creation, if you even need to show that)

Reply:
Removed Worker/Client code.
Have a readme in source file for users to start Workflow from the CLI.
Have the Worker in seperate file.